examples/3dsphere/solid_body_rotation_3d.jl

```diff
@@ -0,0 +1,247 @@
+push!(LOAD_PATH, joinpath(@__DIR__, "..", ".."))
+
+using Test
+using StaticArrays, IntervalSets, LinearAlgebra, UnPack
+
+import ClimaCore:
+    ClimaCore,
+    slab,
+    Spaces,
+    Domains,
+    Meshes,
+    Geometry,
+    Topologies,
+    Spaces,
+    Fields,
+    Operators
+using ClimaCore.Geometry
+
+using Logging: global_logger
+using TerminalLoggers: TerminalLogger
+using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
+
+global_logger(TerminalLogger())
+
+const R = 6.4e6 # radius
+const Ω = 7.2921e-5 # Earth rotation (radians / sec)
+const z_top = 3.0e4 # height position of the model top
+const grav = 9.8 # gravitational constant
+const p_0 = 1e5 # mean sea level pressure
+
+const R_d = 287.058 # R dry (gas constant / mol mass dry air)
+const T_tri = 273.16 # triple point temperature
+const γ = 1.4 # heat capacity ratio
+const cv_d = R_d / (γ - 1)
+
+const T_0 = 300 # isothermal atmospheric temperature
+const H = R_d * T_0 / grav # scale height
+
+# set up function space
+function sphere_3D(
+    R = 6.4e6,
+    zlim = (0, 30.0e3),
+    helem = 4,
+    zelem = 15,
+    npoly = 5,
+)
+    FT = Float64
+    vertdomain = Domains.IntervalDomain(
+        Geometry.ZPoint{FT}(zlim[1]),
+        Geometry.ZPoint{FT}(zlim[2]);
+        boundary_tags = (:bottom, :top),
+    )
+    vertmesh = Meshes.IntervalMesh(vertdomain, nelems = zelem)
+    vert_center_space = Spaces.CenterFiniteDifferenceSpace(vertmesh)
+
+    horzdomain = Domains.SphereDomain(R)
+    horzmesh = Meshes.EquiangularCubedSphere(horzdomain, helem)
+    horztopology = Topologies.Topology2D(horzmesh)
+    quad = Spaces.Quadratures.GLL{npoly + 1}()
+    horzspace = Spaces.SpectralElementSpace2D(horztopology, quad)
+
+    hv_center_space =
+        Spaces.ExtrudedFiniteDifferenceSpace(horzspace, vert_center_space)
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
+    return (hv_center_space, hv_face_space)
+end
+
+# geopotential
+Φ(z) = grav * z
+
+# pressure
+function pressure(ρ, e, normuvw, z)
+    I = e - Φ(z) - normuvw^2 / 2
+    T = I / cv_d + T_tri
+    return ρ * R_d * T
+end
+
+# initial conditions for density and total energy
+function init_sbr_thermo(z)
+
+    p = p_0 * exp(-z * grav / R_d / T_0)
```

Comment:
```suggestion
    p = p_0 * exp(-z / H)
```
Or you can also remove const H?